components/Common/DarkModeToggle/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useTheme } from 'next-themes';
+import { FormattedMessage } from 'react-intl';
+import {
+  MdLightMode as IconLightMode,
```

Comment:
Do we need to import with `as`, the current names sound good enough, no?

Reply:
`MdNightLight` was bit confusing. whether it's a night/light, I thought it would be better to rename it 🤔

Reply:
I guess it's better to keep the original names just for reference. Otherwise if we start arbitrarily changing names well, that's actually confusing.